.travis.yml

```diff
@@ -71,53 +71,56 @@ matrix:
         - NODE_ENV=test NODE_CONFIG_DIR='./config' node "./backend/3drepo.js" &
         - cd ./frontend
        #- unbuffer yarn test
-     #test (v5 be)
-    - language: node_js
-      dist: focal
-      name: Backend tests  (V5)
-      addons:
-        apt:
-         packages:
-            - rabbitmq-server
-      git:
-      submodules: false
-      depth: 1
-      node_js:
-        - "14.17.1"
-      before_install:
-        - wget $SSL_INSTALLER
-        - sudo apt install $PWD/libssl1.0.0_1.0.2n-1ubuntu5.7_amd64.deb
-      script:
-        - cd backend
-        - yarn install
-        - yarn test:v5
-    #test (v5 fe)
-    - language: node_js
-      dist: focal
-      name: Frontend tests (V5)
-      git:
-      submodules: false
-      depth: 1
-      node_js:
-        - "14.17.1"
-      script:
-        - cd frontend
-        - yarn install
-        - yarn test
-    #make sure we're not merging DevOps changes
-    - language: node_js
-      dist: focal
-      name: DevOps Sanity Check
-      addons:
-      git:
-      submodules: false
-      depth: 1
-      node_js:
-        - "14.17.1"
-      before_install:
-        - sudo wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/v4.17.2/yq_linux_amd64 ; sudo chmod +x /usr/local/bin/yq
-      script:
-        - if ( $(yq eval '.variables.customHelmOverride | contains("rabbitmq")' .azure/build-and-deploy.yaml) === "true" ); then echo $(yq eval '.variables.customHelmOverride' .azure/build-and-deploy.yaml); travis_terminate 1;  fi
+
```

Comment:
just delete them 

if we need to go back github has history.

Reply:
Linter test should be removed as well

Reply:
![image](https://user-images.githubusercontent.com/11945337/201398985-7d490f5e-98a9-488a-a1bf-cc7653bedf89.png)
this is failing, may as well fix it here?

Reply:
Also could we setup email notifications like travis used to do on failures?

It's useful to get those

Reply:
addressed the changes

I'm getting lots of emails about it so I think they're going to the committer already.

<img width="658" alt="image" src="https://user-images.githubusercontent.com/23369849/201640170-1a6eef4d-4c8b-4eaa-9040-9ca4f021d304.png">